source/administration/batch-framework.rst

```diff
@@ -0,0 +1,153 @@
+.. _minio-batch-framework:
+
+===============
+Batch Framework
+===============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. versionchanged:: MinIO RELEASE.2022-10-08T20-11-00Z or later
+
+   The Batch Framework was introduced with the ``replicate`` job type in the :mc:`mc` RELEASES.2022-10-08T20-11-00Z.
+
+Overview
+--------
+
+The MinIO Batch Framework allows you to create, manage, monitor, and execute jobs using a YAML-formatted job definition file.
+The batch jobs run directly on the MinIO deployment to take advantage of the deployment's server-side processing power without constraints of the local machine where you run the :ref:`MinIO Client <minio-client>`.
+
+A batch file can define any number of job tasks.
+MinIO does not limit on the number of job tasks that you can define in a batch file.
+
+Once started, MinIO begins processing the job immediately, depending on resources available to the deployment.
```

Comment:
"immediately...depending" feels a little conflicting.

I am guessing it starts any and all jobs specified, and the completion time of those jobs depends on the available resources?

Reply:
Yeah, cleaned up.